wire-ios/Wire-iOS/Sources/Authentication/Coordinator/AuthenticationCoordinator.swift

```diff
@@ -502,32 +506,41 @@ extension AuthenticationCoordinator {
     /// Signs the current user out with a warning.
     private func signOut(warn: Bool) {
         if warn {
-            let signOutAction = AuthenticationCoordinatorAlertAction(
-                title: L10n.Localizable.General.ok,
-                coordinatorActions: [
-                    .showLoadingView,
-                    .signOut(
-                        warn: false
-                    )
-                ],
-                style: .destructive
+            let yesAction = AuthenticationCoordinatorAlertAction(
+                title: L10n.Localizable.Self.Settings.AccountDetails.LogOut.EraseData.Alert.yes,
+                coordinatorActions: [.showLoadingView, .deleteSession(eraseData: true)],
+                style: .default
+            )
+
+            let noAction = AuthenticationCoordinatorAlertAction(
+                title: L10n.Localizable.Self.Settings.AccountDetails.LogOut.EraseData.Alert.no,
+                coordinatorActions: [.showLoadingView, .deleteSession(eraseData: false)],
+                style: .default
             )
 
             let alertModel = AuthenticationCoordinatorAlert(
-                title: L10n.Localizable.Self.Settings.AccountDetails.LogOut.Alert.title,
-                message: L10n.Localizable.Self.Settings.AccountDetails.LogOut.Alert.message,
-                actions: [.cancel, signOutAction]
+                title: L10n.Localizable.Self.Settings.AccountDetails.LogOut.EraseData.Alert.title,
+                message: nil,
```

Comment:
suggestion: keep title and message. 
title: repeating the intention of user Logout
message: explaining the choices: Delete data...